Modules/Remote/RTK.remote.cmake

```diff
@@ -42,5 +42,5 @@ itk_fetch_module(
   "Reconstruction Toolkit (RTK) https://www.openrtk.org/"
   MODULE_COMPLIANCE_LEVEL 3
   GIT_REPOSITORY https://github.com/RTKConsortium/RTK.git
-  GIT_TAG ff1967638805ff73d028fde036bfd81954923ccf
+  GIT_TAG 94d070b0101e4f0a5a7d0649a4559b8ee6d22d63
   )
```

Comment:
Please consider putting a remote module update in a separate pull request. Otherwise, it's just not clear to me why it is part of a "apply clang format" refactoring. Then please explain!

Reply:
Yes.  I plan do create a separate PR for updating remotes.  I'm working with the developers to get fixes in upstream that will make a clean build.